cmd/kueue/main.go

```diff
@@ -274,11 +274,15 @@ func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manag
 func setupProbeEndpoints(mgr ctrl.Manager) {
 	defer setupLog.Info("Probe endpoints are configured on healthz and readyz")
 
-	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
+	readyz := func(req *http.Request) error {
+		return mgr.GetWebhookServer().StartedChecker()(req)
+	}
+
+	if err := mgr.AddHealthzCheck("healthz", readyz); err != nil {
```

Comment:
why change the `healthz`?

Reply:
I'm not sure the change to healthz is required. I will test if this can be simplified. Do you know what healthz is used for?

Reply:
it should indicate if the container is running, https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#types-of-probe

Reply:
Good call, reverted setting it, going to test.